sql/driver/conn.go

```diff
@@ -73,13 +73,8 @@ func (c *conn) Query(stmt string, args []driver.Value) (*rows, error) {
 		case string:
 			param.StringVal = &value
 		case time.Time:
-			// TODO(vivek): pass in time as an input that can be interpreted
-			// by the server.
-			time, err := value.MarshalBinary()
-			if err != nil {
-				return nil, err
-			}
-			param.BytesVal = time
+			t := Datum_Time{Sec: value.Unix(), Nsec: uint32(value.Nanosecond())}
+			param.TimeVal = &t
```

Comment:
We're not sending the location here. Is that unimportant? Probably worth a comment if it isn't.